core/src/components/picker-column/picker-column.tsx

```diff
@@ -70,7 +72,7 @@ export class PickerColumn implements ComponentInterface {
   /**
    * Emitted when the value has changed.
    */
-  @Event() ionChange!: EventEmitter<PickerColumnItem>;
+  @Event() ionChange!: EventEmitter<string | number | undefined>;
```

Comment:
I changed this to get the app to compile, but it likely needs more changes. It may changed based on my keyboard entry work in a future PR.